opentracing-api/pom.xml

```diff
@@ -31,6 +31,14 @@
         <main.basedir>${project.basedir}/..</main.basedir>
         <main.java.version>1.6</main.java.version>
         <main.signature.artifact>java16</main.signature.artifact>
+        <findbugs.jsr305.version>3.0.2</findbugs.jsr305.version>
     </properties>
 
+    <dependencies>
+        <dependency>
+            <groupId>com.google.code.findbugs</groupId>
```

Comment:
Can we do this in a separate PR? It might apply also at different places

Reply:
Do you mean to split adding jsr305 and marking `Tracer.extract` into two separate PRs?